internal/runners/show/show.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/ActiveState/cli/internal/logging"
 	"github.com/ActiveState/cli/internal/output"
 	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/runners/platforms"
```

Comment:
Runners are endpoints, they should never be imported outside of the cmdlet package